components/automate-cli/pkg/verifyserver/models/batchcheck.go

```diff
@@ -123,6 +123,24 @@ func appendCertsByIpToNodeCerts(certsByIP *[]config.CertByIP, ipList []string, p
 
 }
 
+func NewConfig() *Config {
+	return &Config{
+		SSHUser: &SSHUser{},
```

Comment:
Instead of initializing here, how about creating the instance at the config parser level? With this approach how the trigger knows the value is nil or empty? We have to handle each cases differently.
CC: @YashviJain01  @kalroy

Reply:
Yes. If we can enforce that the struct can be created only by this function then it is fine. But if not, we should move it to individual functions in the parser so that we do not get null pointer exception

Reply:
@Dmaddu what do you suggest?